mysql/datadog_checks/mysql/mysql.py

```diff
@@ -94,22 +97,27 @@ def get_library_versions(cls):
         return {'pymysql': pymysql.__version__}
 
     def check(self, _):
+        tags = list(self.config.tags)
```

Comment:
Note that I've changed the tags to pass a mutable local copy for this check run. The only issue is that tags are initialized on the QueryManager. I will make a separate PR which allows passing tags to the QueryManager.execute() function.

Reply:
This has been added: https://github.com/DataDog/integrations-core/pull/8282/files#diff-4ecc52a59650f90cba9b1b1b3a145f8dc3457d133fc52419846c63e62bbe9bd8R126